packages/nextra-theme-docs/src/components/navbar.tsx

```diff
@@ -127,47 +127,51 @@ export function Navbar({ items }: NavBarProps): ReactElement {
             {renderComponent(themeConfig.logo)}
           </div>
         )}
-        {items.map(pageOrMenu => {
-          if (pageOrMenu.display === 'hidden') return null
+        <div className="_flex _gap-4 _overflow-x-auto nextra-scrollbar _py-1.5">
```

Comment:
I noticed that the focus ring styles of the first and last links are being cut off.

![image](https://github.com/user-attachments/assets/39c766c2-fa79-46f7-a243-3ccd330a92be)

Reply:
yeah, noticed it too, little regression